mRemoteNG/mRemoteNG.csproj

```diff
@@ -7,7 +7,7 @@
     <ApplicationIcon>Icons\mRemoteNG.ico</ApplicationIcon>
     <Version>1.77.3</Version>
     <Description>Multi-protocol remote connections manager</Description>
-    <Copyright>2022 mRemoteNG Dev Team, 2010-2013 Riley McArdle, 2007-2009 Felix Deimel</Copyright>
+    <Copyright>2023 mRemoteNG Dev Team, 2010-2013 Riley McArdle, 2007-2009 Felix Deimel</Copyright>
```

Comment:
Hi!
The current year can be replaced with 
```
$([System.DateTime]::Now.Year)
```
so the copyright line would be
```
<Copyright>$([System.DateTime]::Now.Year) mRemoteNG Dev Team, 2010-2013 Riley McArdle, 2007-2009 Felix Deimel</Copyright>
```
This way it will always be up 2 date.

Best,
Simo

Reply:
but that only flat file, so its will not able to show up this properly anyway

Reply:
yes, good idea as it will show up properly from About, but as per flat file will stay weird, but who will read it in plain text :)

Reply:
Guess it's part of _quality of dev life_ 😄  MSBuild got a lot smarter in the last years (.NET core and NuGet)